external/Kconfig

```diff
@@ -1,33 +1,34 @@
 
-config DM
-	bool "DM"
-	default n
-	depends on NET
-	---help---
-		enable the DM functionality
-if DM
 config LWM2M_WAKAAMA
```

Comment:
We want to write an application which uses wakama but we don't want to build DM. 
In this case, current dependency between wakama and DM can be a problem.
I think wakama should be independent with DM because it can always be a problem in situation like ours.